apps/video/js/video.js

```diff
@@ -108,6 +108,27 @@ var loadingChecker =
   new VideoLoadingChecker(dom.player, dom.inUseOverlay, dom.inUseOverlayTitle,
                           dom.inUseOverlayText);
 
+var lastPlayTime = 0;
+
+// Workaround to fix issue reported in Bug 1214157
+// We would like to pause video playback when interrupted by competing audio.
+// Audio Channel API throws interruptbegin event after play event (Bug 1192748)
+// If performance.now() - lastPlayTime is small number we ignore interruptbegin
+// and pause video playback only when interrupted by competing audio.
+// This workaround should be removed with the fix of Bug 1206581
+dom.player.addEventListener('play', function() {
+  lastPlayTime = window.performance.now();
+});
+
+dom.player.addEventListener('mozinterruptbegin', function() {
+  // Ignore mozinterruptbegin event if it immediately follows play event
+  if ((window.performance.now() - lastPlayTime) > 50) {
+    // Pause video when interrupted by competing audio.
+    pause();
+  }
+  lastPlayTime = 0;
```

Comment:
It seems to me 'lastPlayTime' should only get reset when the code concludes there has been an interruption due to competing audio. The way the code is, when 'play' is pressed, 'lastPlayTime' will get reset. I don't think this will cause a problem because when 'mozinterruptbegin' is fired in response to competing audio, 'now - lastPlayTime' is expected to be greater than 50ms even if 'lastPlayTime' didn't get reset. 

It just seems from a purely logical perspective, 'lastPlayTime' isn't being reset in the best place. 

With all that said, I consider this a nit because I don't see any problem with the functionality and this code is intended to be temporary.